Olapic/PublicDocs
# ImageAPI - include_username Updates

What does this Pull Request Do?
------------------------------
Adds in the documentation for the new parameter `include_username` for the image api documentation. It also adds the pictures that are required by this new documentation.

How should this be manually tested?
-----------------------------------
```
cd ~/Olapic/PublicDocs
git checkout gh-pages
jekyll serve
```

visit http://127.0.0.1:4000/articles/image-api.html to ensure the new parameter is available

Any background context you want to provide?
-------------------------------------------
N/a

What are the relevant tickets?
------------------------------
[CDEV-819](https://photorank.atlassian.net/browse/CDEV-819)
